src/app/components/PairSelector.tsx

```diff
@@ -189,7 +190,7 @@ export function PairSelector() {
           const [pair1, pair2] = getPairs(option["name"]);
 
           return (
-            <>
+            <React.Fragment key={`fragment-${index}`}>
```

Comment:
```suggestion
            <React.Fragment key={`pair-${index}`}>
```
If we use fragment for some other indexed lists on the same page, we will have the same error, pair- prefix would be less generic.